config/initializers/govuk_components.rb

```diff
@@ -0,0 +1,4 @@
+Govuk::Components.configure do |config|
```

Comment:
Could you add a TODO to remove these once govuk components gem has been updated and maybe link to the issue to raised on govuk components repo.